src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -1,8 +1,6 @@
 package com.minecolonies.coremod.tileentities;
 
-import com.minecolonies.api.util.InventoryFunctions;
-import com.minecolonies.api.util.InventoryUtils;
-import com.minecolonies.api.util.LanguageHandler;
+import com.minecolonies.api.util.*;
 import com.minecolonies.api.util.Utils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.Utils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)